contrib/debian/netdata.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=netdata real-time system monitoring
+After=network.target
+
+[Service]
+Type=simple
+EnvironmentFile=-/etc/default/netdata
+ExecStart=/usr/sbin/netdata -nd $EXTRA_OPTS
+TimeoutStopSec=30
+Restart=5s
```

Comment:
I must admit, I made a mistake here. `Restart=5s`is totally wrong, the correct lines look like this:
```
Restart=always
RestartSec=5
```